### DISCUSSION TITLE
kann jemand ein paar Tipps geben ? :)

### DISCUSSION THREAD

**Discussion by cutec-chris** [jomjol/AI-on-the-edge-device#1332] [General]
![grafik](https://user-images.githubusercontent.com/5136391/201398519-b0994319-12e1-407b-91f2-22014606b4aa.png)

Was sollt ich ier ändern damit das funktioniert ?

**Comment by jomjol** [comment]
1) Bildqualität verbessern (Schärfe, Kontrast). Selbst das Auge hat schon Probleme, manche Ziffer zu erkennen, dann wird ein neuronales Netz auch nicht funktionieren
2) ROIs korrekt einstellen: https://github.com/jomjol/AI-on-the-edge-device/wiki/ROI-Configuration

**Reply by cutec-chris** [reply]
Danke für den Link, die letzten 10 readings waren alle perfekt. 
Ich hab das Netz allerdings auf das eingesetellt das die Ziffern nicht komplett haben mag, und die Beeiche sind jetzt etwas größer als beschrieben. 

Super Projekt, war schon von der Qualität der Bilder begeistert, das die Erkennung jetzt wirklich funktioniert ist auf einem so kleinen device Wahnsinn.

![grafik](https://user-images.githubusercontent.com/5136391/201424505-35edb8d3-0963-4a0b-a4d9-9a1b6833db26.png)

**Comment by zumheulen** [comment]
Cut a piece of dark paper or cardboard and place it on the unimportant spots in the water meter so that the flashlight doesn't bother you so much. A layer or two of matte tape in front of the flash will also diffuse the light.
